recipes/lens-vpr/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "lens-vpr" %}
+{% set version = "0.1.0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/lens-vpr-{{ version }}.tar.gz
+  sha256: 5e706367882ff2701e2de708a46a7774f457f1d3d793672712e0b7fea510dbf5
+
+build:
+  noarch: python
+  script: {{ PYTHON }} -m pip install . -vv --no-deps --no-build-isolation
+  number: 0
+
+requirements:
+  host:
+    - python >=3.6,<3.12
+    - pip
+  run:
+    - python >=3.6,<3.12
+    - pytorch >=2.1.1
+    - torchvision >=0.16.1
+    - numpy >=1.26.2
+    - pandas >=2.1.1
+    - tqdm >=4.65.0
+    - prettytable >=3.5.0
+    - scikit-learn >=1.2.2
+    - sinabs >=2.0.0
+    - h5py >=3.10.0
+    - imageio >=2.34.1
+    - matplotlib-base >=3.8.2
+    - pynmea2 >=1.19.0
+    - scipy >=1.11.4
+    - seaborn >=0.13.2
+    - wandb >=0.16.2
+
+test:
+  imports:
+    - lens
+  commands:
+    - python -c "import lens"
```

Comment:
I don't feel strongly about this but this test is exactly the same as above and you don't need both.

```suggestion
```